<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeViewer</class>
 <widget class="QWidget" name="VolumeViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="loadLiverBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>载入肝脏</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadGallBtn">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>载入胆囊</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadVesselBtn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>载入血管</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QVTKWidget" name="ivtkWidget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
